filetree/CMakeLists.txt

```diff
@@ -62,8 +62,10 @@ if(WIN32)
 else(WIN32)
     set(path_separator_backslash_in_cmd_line_regex "^[ \t\n\r]*$")
 endif(WIN32)
+string(ASCII 36 Dollar)
 new_ec_test_full_ec_file_path(path_separator_backslash_in_cmd_line
-    path_separator.in "${CMAKE_CURRENT_SOURCE_DIR}\\\\path\\\\separator"
+    path_separator.in
+    "${CMAKE_CURRENT_SOURCE_DIR}\\path\\separator"
```

Comment:
Is there a reason to remove escaping blackslashes here?

Reply:
Yes - this was one of the key changes.  The `add_test(NAME ...)` form apparently does not collapse the backslashes.

Reply:
Yes, you are right on this point